.pipelines/singletenancy/dualstack-overlay/dualstackoverlay-e2e.jobs.yaml

```diff
@@ -0,0 +1,201 @@
+              clusterName: ${{ parameters.clusterName }}-$(commitID)
+              os: linux
+              scaleup: 100
+
+      - job: windows_nodepool
+        displayName: Add Windows Nodepool
+        dependsOn: ${{ parameters.name }}_linux
+        pool:
+          isCustom: true
+          type: linux
+          name: $(BUILD_POOL_NAME_DEFAULT)
+          demands:
+          - agent.os -equals Linux
+          - Role -equals $(CUSTOM_E2E_ROLE)
+        variables:
+          ob_outputDirectory: $(Build.ArtifactStagingDirectory)/output
+          ob_git_checkout: true
+        steps:
+          - checkout: ACNReviewChanges
+            clean: true
+
+          - task: AzureCLI@2
+            inputs:
+              azureSubscription: $(BUILD_VALIDATIONS_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              workingDirectory: $(ACN_DIR)
+              addSpnToEnvironment: true
+              inlineScript: |
+                set -e
+                make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(commitID)
+                make -C ./hack/aks windows-nodepool-up AZCLI=az SUB=$(SUB_AZURE_NETWORK_AGENT_BUILD_VALIDATIONS) CLUSTER=${{ parameters.clusterName }}-$(commitID) VM_SIZE_WIN=${{ parameters.vmSize }}
+                echo "Windows nodes have been successfully added to DualStack Overlay Cluster"
+                kubectl cluster-info
+                kubectl get node -owide
+                kubectl get po -owide -A
+            name: "Add_Windows_Node"
+            displayName: "Add windows node"
+
+      - job: ${{ parameters.name }}_windows
+        displayName: DualStack Overlay Test Suite | Windows - (${{ parameters.name }})
+        timeoutInMinutes: 120
+        dependsOn: windows_nodepool
+        pool:
+          isCustom: true
+          type: linux
+          name: $(BUILD_POOL_NAME_DEFAULT)
+          demands:
+          - agent.os -equals Linux
+          - Role -equals $(CUSTOM_E2E_ROLE)
+        variables:
+          ob_outputDirectory: $(Build.ArtifactStagingDirectory)/output
+          ob_git_checkout: true
+        steps:
+          - checkout: ACNReviewChanges
+            clean: true
+
+          - template: dualstackoverlay-e2e.steps.yaml@ACNTools
+            parameters:
+              name: ${{ parameters.name }}
+              clusterName: ${{ parameters.clusterName }}-$(commitID)
+              os: windows
+              scaleup: 50
+
+      - template: ../../cni/k8s-e2e/k8s-e2e.jobs.yaml@ACNTools
+        parameters:
+          sub: $(BUILD_VALIDATIONS_SERVICE_CONNECTION)
+          clusterName: ${{ parameters.clusterName }}-$(commitID)
+          os: linux
+          dependsOn: ${{ parameters.name }}_windows
+          dualstack: true
+          dns: true
+          portforward: true
+          hostport: true
+          service: true
+
+      - template: ../../cni/k8s-e2e/k8s-e2e.jobs.yaml@ACNTools
```

Comment:
There is an ongoing bug with k8s 1.30 when using this testing suite. #3011 is addressing it for now within the current files on master. Expect this to fail similarly.

Reply:
Sounds good. I'll keep an eye on it.